sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -528,6 +529,10 @@ abstract class RowNumberLike extends AggregateWindowFunction {
   override val updateExpressions: Seq[Expression] = rowNumber + one :: Nil
 }
 
+trait SpecifiedFrameAggregateWindowFunction extends AggregateWindowFunction {
+  override val frame: WindowFrame = SpecifiedWindowFrame(RowFrame, UnboundedPreceding, CurrentRow)
+}
```

Comment:
It seems we no need this trait. If we want window function support using specified window frame, we just do like this:
`override val frame: WindowFrame = UnspecifiedFrame`